.travis.yml

```diff
@@ -4,9 +4,9 @@ sudo: false
 matrix:
   include:
     - go: 1.11.x
-      env: GO111MODULE=off
+      env: GO111MODULE=on
     - go: 1.12.x
-      env: GO111MODULE=off
+      env: GO111MODULE=on
```

Comment:
Yeah, I discussed this with @jhump offline. We decided that since this is intended as a command line tool and not a library, this would be fine to turn on. Also, I'm planning on releasing some binaries with the goreleaser changes I made here: #90